src/Fetcher/ConfTechFetcher.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Fetcher;
+
+use App\Entity\Conference;
+use Gedmo\Sluggable\Util\Urlizer;
+use GuzzleHttp\Client;
+use GuzzleHttp\Exception\GuzzleException;
+use Psr\Log\LoggerInterface;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+class ConfTechFetcher implements FetcherInterface
+{
+    const SOURCE = 'conf-tech';
+    private $em;
+    private $repository;
+    private $logger;
+
+    public function __construct(RegistryInterface $doctrine, LoggerInterface $logger)
+    {
+        $this->em = $doctrine->getManager();
+        $this->logger = $logger;
+        $this->repository = $this->em->getRepository(Conference::class);
+    }
+
+    public function getUrl(array $params): string
+    {
+        return "https://raw.githubusercontent.com/tech-conferences/conference-data/master/conferences/$params[date]/$params[tag].json";
+    }
+
+    public function fetch(): array
+    {
+        $params = [
+            '2018' => [
+                'php',
+                'javascript',
+                'golang',
+            ],
+            '2019' => [
+                'php',
+                'ruby',
+                'css',
+            ],
+        ];
+
+        $conferencesFinal = [];
+
+        foreach ($params as $date => $technologies) {
+            foreach ($technologies as $technologie) {
+                $client = new Client();
+                try {
+                    $response = $client->request('GET', $this->getUrl(['date' => $date, 'tag' => $technologie]));
+                } catch (GuzzleException $e) {
+                    $this->logger->error($e->getMessage());
+                }
+
+                $fetchedConferences = json_decode($response->getBody());
+
+                $newConferencesCount = 0;
+                $source = self::SOURCE;
+
+                $conferenceTotal = [];
+
+                foreach ($fetchedConferences as $fetchedConference) {
```

Comment:
All this treatment should be moved in another function. To be unit testable in the future